### PR TITLE
view: log app-id on map and unmap

### PR DESCRIPTION
**Pull request by alex-courtis**
There is no means to determine app-id, as there is no wayland equivalent of xprop.

Log app-id as well as title.

### CODE REVIEW

File: river/View.zig [hunk 1]
```diff
@@ -480,7 +480,7 @@ pub fn shouldTrackConfigure(self: Self) bool {
 
 /// Called by the impl when the surface is ready to be displayed
 pub fn map(self: *Self) !void {
-    log.debug("view '{s}' mapped", .{self.getTitle()});
+    log.debug("view app-id='{s}' title='{s}' mapped", .{self.getAppId(), self.getTitle()});
```
Comment (alex-courtis) [inline river/View.zig:483]: We could change this to info to be a bit more user friendly.

**Comment by ifreund** [comment]
[lswt](https://git.sr.ht/~leon_plickat/lswt) can be used to determine app-id and other properties.

**Comment by Leon-Plickat** [comment]
Also note that this is a bit flaky, as some applications unfortunately set their app-id *after* they map their toplevels.

**Comment by alex-courtis** [comment]
> [lswt](https://git.sr.ht/~leon_plickat/lswt) can be used to determine app-id and other properties.

Fantastic! OK if I add that to the man page @Leon-Plickat ?

**Comment by alex-courtis** [comment]
> Also note that this is a bit flaky, as some applications unfortunately set their app-id _after_ they map their toplevels.

That's unfortunate. At least we will see it on unmap.

**Comment by alex-courtis** [comment]
lswt works beautifully.

For the case of badly behaved apps (like zoom) that quickly open and close "notification" windows the user can do something like:

```sh
while true; do lswt; echo; sleep 0.5; done
```

Thanks for the review, closing.

**Pull request closed without merging**